ecosystem/sep-0024.md

```diff
@@ -227,7 +280,8 @@ Name | Type | Description
 -----|------|------------
 `asset_code` | string | The code of the stellar asset the user wants to receive for their deposit with the anchor. The value passed must match one of the codes listed in the [/info](#info) response's deposit object.
```

Comment:
Lets add that this must match the `buy_asset` from the quote, if `quote_id` is specified. Same for the issuer. Don't forget to also update the withdraw endpoint in the same way.